app/controllers/lists_controller.rb

```diff
@@ -122,7 +122,8 @@ def update_list_items
 
     if save_list_items
       ListPublisher.new(@tag).perform
-      flash[:notice] = "Items added to list successfully"
+      new_list_item_count = edit_list_items_params.count
+      flash[:notice] = "#{new_list_item_count} #{'link'.pluralize(new_list_item_count)} successfully added to the list"
```

Comment:
```suggestion
      flash[:notice] = "#{'pluralize(edit_list_items_params.count, 'link')} successfully added to the list"
```

minor, this seems a little cleaner and save the need for an extra variable.

Reply:
I did think about using this, but it requires inclusion of the `ActionView::Helpers::TextHelper` module. Do you think that's preferable?

Reply:
Oh do we not have that already included in order to be able to call `pluralize` on a string? I think you can call helpers from a controller by doing `helpers.pluralize` with the helpers object having access to all your view helper methods (https://www.bigbinary.com/blog/rails-add-helpers-method-to-ease-usage-of-helper-modules-in-controllers)